map_test.go

```diff
@@ -75,6 +75,10 @@ func TestMap(t *testing.T) {
 		t.Error("Want value 42, got", v)
 	}
 
+	sliceVal := make([]uint32, 1)
+	qt.Assert(t, m.Lookup(uint32(0), sliceVal), qt.IsNil)
```

Comment:
Hmm, this is a lookup in a normal array map right? Why does looking up a slice work here?

Reply:
it eventually makes it to `unsafeBackingMemory()`

The key/value of map can be a non-scalar type, I thought I'd check it works with slices here. I thought `uint64[2]` would be a valid `value` in the kernel?

Reply:
Ah yes, true. Shows how finicky this all is :(